lesson09/script.js

```diff
@@ -0,0 +1,46 @@
+function showLoadingImg(){
+    const loadingImg = document.createElement('img');
+    loadingImg.src = "loading-circle.gif";
+    loadingImg.alt = "ローディング画像";
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoadingImg(){
+    document.getElementById('js-loading').remove();
+}
+
+ function showCategories(result){
+    const fragment = document.createDocumentFragment();
+    for(const list of result){
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+        a.textContent = list.text;
+        a.href = `/${list.to}`;
+        img.src = list.img;
+        img.alt = list.alt;
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    document.getElementById('js-list').appendChild(fragment);
+}
+
+async function SetCategoriesAfter3Seconds(){
```

Comment:
I'm not using await only in async. I don't need this function here.

Reply:
Thank you for your comment!
Certainly I don't need async except when using await.
I will delete "async" in this function.